### ISSUE TITLE
Add echo entity to fieldmap sequence with multiple phase-encoded directions

### ISSUE THREAD

**Issue by witherscp** [bids-standard/bids-specification#1455]
### Your idea

I have fieldmap sequences collected in the reverse and forward directions to use for correcting fMRI resting state sequences. The reverse/forward fieldmap sequences are multi-band (3 different echo times) corresponding to fMRI resting state epi sequences which are also multi-band. In the [current bids specification](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#case-4-multiple-phase-encoded-directions-pepolar), I do not see an option to add the echo-index entity to the fieldmap with multiple phase-encoded direction files. I only see the option to add an entity for multiple runs. Would you consider adding the echo entity to this file type? @ericearl and others may be able to assist.

Thanks!

**Comment by effigies** [comment]
There seems to be a confusion here about the difference between multiband and multiecho. These are two different concepts that can be used together or separately.

Not sure how I feel about adding `echo-` to `epi` fieldmaps. On the one hand, people do acquire the data; on the other hand, you generally just calculate the fieldmap based on the shortest echo, as distortion is identical across echos and dropout is minimized in the shortest echo. From the perspective of maintaining fieldmap estimation tools, I would worry about users adding poor SDC correction specifications to the data on the grounds of "well, I collected these files to correct those".

**Comment by tsalo** [comment]
I agree that the main thing that would prevent us from adding this to BIDS would be how it might lead to bad distortion correction. Here's a thread that talked about this a bit: https://neurostars.org/t/multi-echo-pepolar-fieldmaps-bids-spec-sdcflows-grayzone/23933.

**Comment by witherscp** [comment]
Thank you for the quick response and for pointing me to the thread that discussed this issue previously. Sorry I am new to fMRI data analysis, and I was not aware that distortion correction was identical across echos. I think it makes most sense to just create a fieldmap file for the shortest echo in my dataset. Thanks for your help!
